openscap_daemon/config.py

```diff
@@ -66,6 +66,12 @@ def __init__(self):
         self.fetch_cve_timeout = 10*60
         self.cve_feed_manager = cve_feed_manager.CVEFeedManager()
 
+        # REST API Section
+        self.api_enabled = False
+        self.api_port = 5000
+        self.api_host = "127.0.0.1"
+        self.api_debug = False
```

Comment:
I suggest to rename these variables to `rest_` to make the code a bit more readable.

Reply:
Changed to rest_*